static/css/devhub/new-landing/common.less

```diff
@@ -119,9 +130,15 @@
     height: 275px;
     position: absolute;
     right: -186px;
-    top: 100px;
+    top: ~"calc(50% - 57px)"; // Top of the image to the cog.
```

Comment:
The cog is this:

<img width="348" alt="_ ____ _" src="https://cloud.githubusercontent.com/assets/1514/21848995/5fff35fa-d7fb-11e6-928a-e97b1e3e2f39.png">

So I'm referring to the top of the image to the cog at the top of the ladder.